Content.Server/Botany/Components/PlantHolderComponent.cs

```diff
@@ -71,6 +71,12 @@ public sealed partial class PlantHolderComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("health")]
     public float Health;
 
+    ///<summary>
+    /// The bonus to potency from fertiliser applied to the curent crop. This is added with the seed's own Potency to determine things like crop reagent contents.
+    ///</summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("potencyBonus")]
```

Comment:
Since this datafield matches the automatically-derived name, the explicit DataField("potencyBonus") can be omitted. I know the code above and below still use the old convention, but let's try to update the convention in the new code.